envs/.env.dev

```diff
@@ -1,11 +1,19 @@
-# This is the standard lower environment for Content API.
+# Domain for the CMS Content API
 NEXT_PUBLIC_DRUPAL_BASE_URL=https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov
+
+# Domain for retrieving CMS images; likely unused
 NEXT_IMAGE_DOMAIN=https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov
 
-# Prod bucket for vets-website assets
-# NEXT_PUBLIC_ASSETS_URL=https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/
-NEXT_PUBLIC_ASSETS_URL=/generated/
+# Standard VA.gov build type
+NEXT_PUBLIC_BUILD_TYPE=vagovdev
+
+# Location to load vets-website assets
+NEXT_PUBLIC_ASSETS_URL=https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/
```

Comment:
This is necessary for vets-website assets to be pulled from their S3 bucket; otherwise, they get pulled from dev.va.gov/generated/, which tries to resolve to the next-build/content-build buckets. This works but it is much slower than sourcing from the vets-website bucket.